### DISCUSSION TITLE
Deprecated Auth0 Instructions. Is there a workaround?

### DISCUSSION THREAD

**Discussion by insanity54** [PostgREST/postgrest#3088] [Q&A]
Hi I'm getting started with PostgREST and I'm looking into setting it up with JWTs generated at Auth0.

The documentation at https://postgrest.org/en/stable/integrations/jwt_gen.html depends on a deprecated Auth0 feature, [Rules and Hooks](https://auth0.com/docs/manage-users/user-accounts/metadata/manage-metadata-rules)

> We plan to deprecate Rules and Hooks in 2024, and they are no longer available to new tenants created as of October 16, 2023.

This seems like a blocker for following the PostgREST docs. Is there a workaround for this situation?

**Comment by laurenceisla** [comment]
Hi, sorry I'm not familiarized with Auth0 new features. In the meantime, I created an Issue in the docs https://github.com/PostgREST/postgrest-docs/issues/715 until a collaborator finds a solution/suggestion to update them.
